shared/util/saga.js

```diff
@@ -132,11 +132,18 @@ function safeTakeEveryPure<A, R, FinalEffect, FinalErrorEffect>(
   })
 }
 
+function* callPromise<Args, T>(fn: (...args: Args) => Promise<T>, ...args: Args): Generator<any, T, any> {
```

Comment:
comment about when to use `callPromise` vs `callUntyped` would probably come in handy

Reply:
i added one in part 2